elevate-cpanel

```diff
@@ -4714,6 +4715,7 @@ Helps to upgrade CentOS 7 cPanel servers to AlmaLinux 8 or Rocky Linux 8.
        --clean                        Cleanup scripts and files created by elevate-cpanel
        --upgrade-to=[rocky|almalinux] Update to AlmaLinux 8 or Rocky Linux 8
                                       [default to 'almalinux']
+       --yes, --non-interactive       Skip the yes/no prompt before proceeding with the upgrade.
```

Comment:
Could we add a '[default to 'no']' here?

Reply:
I'm also not sure if we need --non-interactive. That was just an idea. Do we really want it?

Reply:
I'm not sure if we need --non-interactive.  Probably just having --yes will suffice.